packages/nx-release-version/index.js

```diff
@@ -78,9 +81,11 @@ module.exports = async function(tree, options) {
     callback: async (tree, options) => {
       const result = await callback(tree, options);
 
-      const versionedFiles = await runSetVersion();
-      if (versionedFiles) {
-        const changedFiles = Array.isArray(result) ? result : result.changedFiles;
+      // Only update artifacts if there were changes
+      const changedFiles = Array.isArray(result) ? result : result.changedFiles;
+      if (changedFiles.length > 0) {
+        fs.writeFile(path.join(REPO_ROOT, '.rnm-publish'), '', () => null);
```

Comment:
This won't get picked up in the release commit right? Because nx is smarter about not committing all files?

Reply:
The release commit deletes the version plans so this should never trigger.

Reply:
I meant more, the step that generates the release commit won't pick up the file before as part of a `git add *` or anything like that I hope? Considering we list our changed files, I think not.

Reply:
This step is after versioning, but before `git add`. We hooked into here to do the additional artifact update.